### PR TITLE
docs: add example for multiple glob dirs

### PR DESCRIPTION
**Pull request by mmkal**

### CODE REVIEW

**Review by papb — COMMENTED** [review]
Can you please remind me how are the migrations ordered by default in this case? Is it lexicographically ascending on the file names directly, or first on the folder names and then file names?

**Comment by mmkal** [comment]
@papb good call-out, I added a section to the docs.

File: README.md [hunk 1]
```diff
@@ -321,6 +321,29 @@ const umzug = new Umzug({
...
+    glob: '{first-folder/*.js,second-folder-with-different-naming-convention/*.js}',
+  },
+  context: new Sequelize(...),
+  logger: console,
+});
+```
+
+Note on migration file sorting:
+
+- file matches, found using [glob](https://npmjs.com/package/glob), will be alphabetically sorted based on their paths
```
Comment (papb) [inline README.md:341]:
```suggestion
- file matches, found using [glob](https://npmjs.com/package/glob), will be lexicographically sorted based on their paths
```

File: README.md [hunk 2]
```diff
@@ -321,6 +321,29 @@ const umzug = new Umzug({
...
+  - so if your migrations are `one/m1.js`, `two/m2.js`, `three/m3.js`, the resultant order will be `one/m1.js`, `three/m3.js`, `two/m2.js`
+  - similarly, if your migrations are called `m1.js`, `m2.js`, ... `m10.js`, `m11.js`, the resultant ordering will be `m1.js`, `m10.js`, `m11.js`, ... `m2.js`
+- The easiest way to deal with this is to ensure your migrations appear in a single folder, and their paths match alphabetically with the order they should run in
```
Comment (papb) [inline README.md:344]:
```suggestion
- The easiest way to deal with this is to ensure your migrations appear in a single folder, and their paths match lexicographically with the order they should run in
```

**Review by papb — CHANGES_REQUESTED** [review]

**Pull request merged**